pr-label-by-review/action.yml

```diff
@@ -22,6 +22,10 @@ inputs:
     description: 'Label to add when PR status is changes requested.'
     default: 'changes requested'
 
+  protectionType:
```

Comment:
Sorry, ik realiseer me dit nu pas, maar deze moet in kebab-case. Plus, graag even de readme bijwerken

Reply:
done